data/inria/preprocess.py

```diff
@@ -42,8 +42,8 @@ def patch_inria(data_dir, save_dir, patch_size, pad, overlap):
     # get rgb and gt files
     for city_name in tqdm(SAVE_CITY, desc='City-wise'):
         for tile_id in tqdm(range(1, 37), desc='Tile-wise', leave=False):
-            rgb_filename = os.path.join(data_dir, 'image', '{}{}.tif'.format(city_name, tile_id))
-            gt_filename = os.path.join(data_dir, 'truth', '{}{}.tif'.format(city_name, tile_id))
+            rgb_filename = os.path.join(data_dir, 'images', '{}{}.tif'.format(city_name, tile_id))
+            gt_filename = os.path.join(data_dir, 'gt', '{}{}.tif'.format(city_name, tile_id))
```

Comment:
Can you verify if they are the name of folders after you download from https://project.inria.fr/aerialimagelabeling/download/

Reply:
Yes, I downloaded the dataset from here today and they are 'images' and 'gt'. Bass connections students also got 'images' and 'gt', I believe.
I downloaded .7z files. Could it lead to different folder names from downloading using the script provided?